registry

```diff
@@ -475,3 +475,32 @@ erm:ERM00000458 a obo:CHEBI_59999 ; rdfs:label "NanoSolveIT Material A21" ; dct:
 erm:ERM00000459 a obo:CHEBI_59999 ; rdfs:label "NanoSolveIT Material 2022 1" ; dct:creator wd:Q61910892 .
 erm:ERM00000460 a obo:CHEBI_59999 ; rdfs:label "NanoSolveIT Material 2022 2" ; dct:creator wd:Q61910892 .
 erm:ERM00000461 a obo:CHEBI_59999 ; rdfs:label "NanoSolveIT Material 2022 3" ; dct:creator wd:Q61910892 .
+
+
+erm:ERM00000462 a obo:CHEBI_59999 ; rdfs:label "POLYRISK material 2022 1" ; dct:creator wd:Q27061849
```

Comment:
@jpmvrijn, when you merge, plz add the missing periods at the end of the lines.

Reply:
sorry for these, added.